### ISSUE TITLE
geonetwork on docker - Error when getting appender named 'fileAppender'

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/georchestra#3149]
Starting the docker composition on master yields:
```
geonetwork_1                | 2020-09-28 09:14:41,419 ERROR [geonetwork] - Error when getting appender named 'fileAppender'. Check your log configuration file. No appender found.
```
I do not understand where GeoNetwork expects to find an appender named as such, since there is none in https://github.com/georchestra/datadir/blob/docker-master/geonetwork/log4j/log4j.xml

**Comment by pmauduit** [comment]
i think that it is used to be able to download logs from the admin ui, there is no such appender in the docker datadir, which should make the feature unavailable in case of docker setups

**Comment by pmauduit** [comment]
indeed: https://github.com/georchestra/geonetwork/blob/georchestra-gn3.8.2/services/src/main/java/org/fao/geonet/api/site/LoggingApi.java#L207-L226
